lib/frontend/input_behavior_default.dart

```diff
@@ -31,10 +31,23 @@ class InputBehaviorDefault extends InputBehavior {
     }
   }
 
+  String? _composingString = null;
+
   @override
   TextEditingValue? onTextEdit(TextEditingValue value, Terminal terminal) {
+    // we just want to detect if a composing is going on and notify the terminal
+    // about it
+    if (value.composing.start != value.composing.end) {
+      _composingString = (_composingString ?? '') + value.text;
```

Comment:
Should this line be:
```
_composingString = value.text
```
?

Reply:
In my machine it looks like this when written as `_composingString = (_composingString ?? '') + value.text;`:

value.text: h
value.text: he
value.text: hel
value.text: hell
value.text: hello

_composingString: h
_composingString: hhe
_composingString: hhehel
_composingString: hhehelhell
_composingString: hhehelhellhello

Reply:
I'm using `Flutter 2.1.0-12.2.pre • channel beta`.

Reply:
Hm, Ok. Yes, seems like a bug.
I was not sure about potential multi-character compose states but seems like my assumptions were incorrect.
Will fix that